kivy/core/window/__init__.py

```diff
@@ -307,7 +307,7 @@ class WindowBase(EventDispatcher):
                 The *unicode* parameter has be deprecated in favor of
                 codepoint, and will be removed completely in future versions.
 
-        `on_dropfile`: str
+        `on_dropfile`: filename (bytes):
```

Comment:
I think this could potentially be either a string or bytes, depending on the OS and window backend. So maybe say it could be either bytes or string?